recipes/argoverse/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "argoverse" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/argoai/{{ name }}-api/archive/master.tar.gz
```

Comment:
```suggestion
  url: https://github.com/argoai/{{ name }}-api/archive/{{ version }}.tar.gz
```

Reply:
Thanks for all the helpful comments, @chrisburr.

With regards to the url, I seem to get a 404 if I use `version` instead of `master`. I don't think that there are any releases on this repository -- the version number can only be found in `setup.py`. What are your thoughts?

Reply:
The upstream repo should have tags that to correspond to releases. Without we can't reliably rebuild old versions or automatically make update PRs using the bots.

Reply:
Sounds good. I'll circle back with you on this one after I ping upstream.